front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -1,31 +1,79 @@
 .time-stops-datasheet {
+  padding-bottom: 16px;
+  border-radius: 10px;
+  box-shadow:
+    0px 1px 0px rgba(255, 255, 255, 1) inset,
+    0px 4px 7px -3px rgba(255, 171, 88, 0.17),
+    0px 2px 4px rgba(0, 0, 0, 0.22);
+  background-color: var(--ambiantB15);
+  --dsg-cell-background-color: var(--white100);
+  --dsg-cell-disabled-background-color: var(--white100);
+  --dsg-header-text-color: var(--grey50);
+  --dsg-header-active-text-color: var(--black100);
+
   .activeRow {
-    font-weight: bold;
-    background-color: var(--coolgray1);
-    --dsg-cell-background-color: var(--coolgray1);
-    --dsg-cell-disabled-background-color: var(--coolgray1);
-    .dsg-cell {
-      font-weight: bold;
+    font-weight: semibold;
+    color: var(--black100);
+    .dsg-cell-gutter,
+    .cell-index {
+      font-weight: normal;
     }
   }
 
+  .padded-header > * {
+    padding-left: 8px;
+    padding-right: 16px;
+  }
+
+  .cell-index {
```

Comment:
Why cell-index for the classname ? Why not .secondary-code-column ?

Reply:
I thought of it as the CH code was acting like a secondary index, like the very first index column, hence their similar color and thus the name.

I am fine with changing it to secondary-code-column though ^^